Modules/IO/NIFTI/test/itkNiftiReadWriteDirectionTest.cxx

```diff
@@ -40,85 +42,86 @@ itkNiftiReadWriteDirectionTest(int ac, char * av[])
   }
 
   using TestImageType = itk::Image<float, 3>;
-  auto reader_lambda = [](std::string filename) -> TestImageType::Pointer {
+  auto ReaderLambda = [](std::string filename) -> TestImageType::Pointer {
     itk::ImageFileReader<TestImageType>::Pointer reader = itk::ImageFileReader<TestImageType>::New();
-    std::string                                  test(filename);
-    reader->SetFileName(test);
+    reader->SetFileName(filename);
     reader->Update();
     return reader->GetOutput();
   };
 
-  using TestImageType = itk::Image<float, 3>;
-  TestImageType::Pointer InputImage = reader_lambda(av[1]);
-  TestImageType::Pointer InputImage_noqform = reader_lambda(av[2]);
-  TestImageType::Pointer InputImage_nosform = reader_lambda(av[3]);
+  TestImageType::Pointer inputImage = ReaderLambda(av[1]);
```

Comment:
why not use `TestImageType::Pointer inputImage = itk::ReadImage<TestImageType>(av[1]);`? ReadImage is now part of C++ ITK.

Reply:
Using ReadImage throughout now

Reply:
:+1: Done in ceac451.